src/WebJobs.Script/ExtensionBundle/ExtensionBundleManager.cs

```diff
@@ -35,6 +33,35 @@ public ExtensionBundleManager(ExtensionBundleOptions options, IEnvironment envir
             _options = options ?? throw new ArgumentNullException(nameof(options));
         }
 
+        public async Task<ExtensionBundleInfo> GetExtensionBundleInfo()
+        {
+            if (IsExtensionBundleConfigured())
+            {
+                if (_extensionBundleVersion == null && TryLocateExtensionBundle(out string path))
+                {
+                    _extensionBundleVersion = Path.GetFileName(path);
```

Comment:
simplify, check for not null first and return ExtensionBundleInfo

Reply:
Have updated the code logic below to simplify a little bit. checking null before hand would require me to call Path.GetFileName(path) twice and wouldn't simplify the code.